boilr.go

```diff
@@ -1,7 +1,18 @@
 package main
 
-import "github.com/tmrts/boilr/pkg/cmd"
+import (
+  "github.com/tmrts/boilr/pkg/boilr"
+  "github.com/tmrts/boilr/pkg/cmd"
+  "github.com/tmrts/boilr/pkg/util/exit"
+  "github.com/tmrts/boilr/pkg/util/osutil"
+)
 
 func main() {
-	cmd.Run()
+  if exists, _ := osutil.DirExists(boilr.Configuration.TemplateDirPath); ! exists {
```

Comment:
Ignoring the returned error will cause some obscure bugs/headaches later on, just use an err check and then an `exit.Error`